backend/src/main/java/gov/cdc/usds/simplereport/api/queue/ApiTestOrderDataResolver.java

```diff
@@ -92,6 +93,11 @@ public DeviceType deviceType(ApiTestOrder apiTestOrder) {
     return apiTestOrder.getWrapped().getDeviceType();
   }
 
+  @SchemaMapping(typeName = "TestOrder", field = "specimenType")
+  public SpecimenType specimenType(ApiTestOrder apiTestOrder) {
+    return apiTestOrder.getWrapped().getSpecimenType();
```

Comment:
specimenType is preloaded when the test order (queue) is loaded from the db by the entity graph. so this will not cause an extra call to the db